libs/ui/carousel/helm/src/lib/hlm-carousel.component.ts

```diff
@@ -63,8 +63,13 @@ export class HlmCarouselComponent {
 	private readonly _canScrollNext = signal(false);
 	public canScrollNext = this._canScrollNext.asReadonly();
 
+	private readonly _currentSlide = signal(0);
+	public currentSlide = this._currentSlide.asReadonly();
+	private readonly _slideCount = signal(0);
+	public slideCount = this._slideCount.asReadonly();
```

Comment:
We could also make these signals readonly to ensure the values won't be unexpectedly overridden:

```suggestion
	private readonly _currentSlide = signal(0);
	public readonly currentSlide = this._currentSlide.asReadonly();
	private readonly _slideCount = signal(0);
	public readonly slideCount = this._slideCount.asReadonly();
```

Reply:
Good idea